ex2.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+
+
+def get_rows_and_columns(matrix):
```

Comment:
The name doesn;t express well enought what the function does, it sound like you are returning the rows and the collumns not the amount of rows and collumns.